### PR TITLE
[biomed nl] multi entity overview

### PR DESCRIPTION
**Pull request by chejennifer**
Add multi entity overview when more than 1 entity detected and no existing props

### CODE REVIEW

**Review by pradh — APPROVED** [review]
Nice!

Add a test?

**Comment by chejennifer** [comment]
Thanks for the review! Added a test case

**Review by pradh — APPROVED** [review]
Thanks for the test

File: server/integration_tests/test_data/e2e_triple/whatvirusspeciesarers13317andrs7903146/chart_config.json [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+        "blocks": [
+          {
+            "columns": [
+              {
+                "tiles": [
+                  {
+                    "answerTableTileSpec": {
+                      "columns": [
+                        {
+                          "header": "->alignmentQuality",
```
Comment (pradh) [inline server/integration_tests/test_data/e2e_triple/whatvirusspeciesarers13317andrs7903146/chart_config.json:15]: future, I assume header will be improved in future?

Comment (chejennifer) [reply server/integration_tests/test_data/e2e_triple/whatvirusspeciesarers13317andrs7903146/chart_config.json:15]: yes, header will be improved in the future!

**Pull request merged**